File: app/src/main/java/bobrchess/of/by/belaruschess/view/activity/UserContractView.java
package bobrchess.of.by.belaruschess.view.activity;

import java.util.List;

import bobrchess.of.by.belaruschess.dto.UserDTO;

public interface UserContractView extends BaseContractView {
    void showUsers(List<? extends UserDTO> users);
    void showUser(UserDTO user);
}

File: app/src/main/java/bobrchess/of/by/belaruschess/view/activity/UserContractView.kt
package bobrchess.of.by.belaruschess.view.activity

import bobrchess.of.by.belaruschess.dto.UserDTO

interface UserContractView : BaseContractView {
    fun showUsers(users: List<UserDTO>?)
    fun showUser(user: UserDTO?)
}